src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = props => {
+  const { todo, onClose } = props;
+  const [isLoading, setIsLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getUser(todo.userId)
```

Comment:
Ensure that `todo.userId` is valid before calling `getUser`. Consider adding a check to handle cases where `userId` might be `null` or `undefined` to prevent potential runtime errors.